src/swr/jwt/useSchemaTree.ts

```diff
@@ -31,81 +25,110 @@ export type SchemaTable = {
   schema_name: string;
   table_name: string;
   table_type: SchemaTableType;
-  quoted_table_name: string;
-  path: string;
-  quoted_path: string;
+  path: string[];
   columns: SchemaTableColumn[];
   is_system_table: boolean;
 };
 
 export type Schema = {
   schema_name: string;
-  quoted_schema_name: string;
-  path: string;
-  quoted_path: string;
+  path: string[];
   tables: SchemaTable[];
 };
 
 export const postFetch = (data: QueryResultSuccess): Schema[] => {
   const constructSchemaTreeFromFlatList = (input: SchemaDescription[]) => {
+    const formatDataType = (dataType: string) => {
+      if (dataType.endsWith('_array')) {
+        const tmp = dataType.split('_');
```

Comment:
Does this break with `geo_point_array` -> `ARRAY(GEO_POINT)` ? 🤔

Reply:
You are right. I will fix this now.

Reply:
This is now done. Thanks for pointing this out.